CHANGELOG.md

```diff
@@ -34,8 +34,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - The class layout of `BufCpu` no longer depends on whether the CUDA and HIP back-ends are enabled. #1612
 - Fixed several smaller bugs in `alpaka::Vec` #1620
 - Destructors no longer throw an exception #1632
+- Implemented work-around for Intel compiler bug with OpenMP back-ends #1677
 
 ### New Features:
+- alpaka now has native complex number support #1336
```

Comment:
Perhaps this should not be the first point? Probably switch to C++17 makes more sense first.

Reply:
I just ordered the points by PR numbers.